trillian_map_api.proto

```diff
@@ -76,6 +76,14 @@ message GetMapLeavesByRevisionRequest {
   int64 revision = 3;
 }
 
+message GetLastInRangeByRevisionRequest {
+  int64 map_id = 1;
+  int64 revision = 2;
+  // [start_index, end_index)
```

Comment:
This could maybe do with a bit more explanation?

Also, it also seems to contradict the statement on `MapLeaf` that "Map entries do not have a well defined ordering"